CurrencyTests/Currency-React-Web-App.Tests/Services.Test/CurrentDateCurrencyServiceTest.cs

```diff
@@ -96,6 +101,7 @@ public void SortByCurrencyFieldName_ShouldSortBySelectedFieldAscending(SortByFie
     [InlineData( SortByFieldEnum.Currency )]
     [InlineData( SortByFieldEnum.Rate )]
     [InlineData( SortByFieldEnum.Text )]
+    [InlineData( SortByFieldEnum.ExchangeDate )]
     public void SortByCurrencyFieldName_ShouldSortBySelectedFieldDescending(SortByFieldEnum field)
     {
         List<CurrentDateCurrencyModel> result =
```

Comment:
This tests should designed easier.

Let's say you want to check descending order


Define: input [might be via inlineData]
Define output (expected value) [might be via inlineData]
Define FieldName type in inlin data
Act: use your logic

Your test method would be very clear like:

var result - _sertService(sth)

Assert.IsEqual(expected, result)